tests/test_roc.py

```diff
@@ -49,6 +49,52 @@ def test_get_bootstrapped_roc_ci_curves():
     assert roc_95.high_az_val >= roc_65.high_az_val
 
 
+def test_average_roc_curves():
+    y_true = np.random.randint(0, 2, 500).astype(np.int)
```

Comment:
As discussed, you might want to fix the random state using a numpy.random.RandomState object and/or use a fixture here.

We use this fixture for test_stats.py:
```python
@pytest.fixture(autouse=True)
def reset_seeds():
    np.random.seed(42)
    yield

```

Reply:
Added fixture in next commit.